live-examples/css-examples/fonts/font-stretch.html

```diff
@@ -0,0 +1,48 @@
+<section id="example-choice-list" class="example-choice-list" data-property="font-stretch">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">font-stretch: condensed;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: expanded;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: ultra-expanded;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 50%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 100%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 150%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+</section>
+
+<div id="output" class="output hidden">
+    <section id="default-example" class="default-example">
+      <div id="example-element">
```

Comment:
The other font examples use `<p>` here, which seems more semantic than a `<div>`. Is there a reason you chose a `<div>`?